auto_nag/user_activity.py

```diff
@@ -4,90 +4,97 @@
 
 from enum import Enum
 
-from libmozdata.bugzilla import Bugzilla, BugzillaUser
+from libmozdata import utils as lmdutils
+from libmozdata.bugzilla import BugzillaUser
 
 from auto_nag import utils
+from auto_nag.people import People
 
 DEFAULT_ACTIVITY_WEEKS = 26
+DEFAULT_ABSENT_WEEKS = 13
 
 
 class UserStatus(Enum):
     UNDEFINED = 1
     DISABLED = 2
     INACTIVE = 3
+    ABSENT = 4
 
 
 class UserActivity:
-    def __init__(self, weeks_count=DEFAULT_ACTIVITY_WEEKS) -> None:
-        self.weeks_count = weeks_count
+    def __init__(
+        self,
+        activity_weeks_count=DEFAULT_ACTIVITY_WEEKS,
+        absent_weeks_count=DEFAULT_ABSENT_WEEKS,
+    ) -> None:
+        self.activity_weeks_count = activity_weeks_count
+        self.absent_weeks_count = absent_weeks_count
+        self.people = People.get_instance()
+
+        self.activity_limit = lmdutils.get_date("today", self.activity_weeks_count * 7)
+        self.seen_limit = lmdutils.get_date("today", self.absent_weeks_count * 7)
 
     def check_users(self, user_emails):
-        user_emails = set(user_emails)
+        user_emails = self.get_not_employees(user_emails)
```

Comment:
Could you add a comment explaining that we do not need to check activity for people that are employees and we just assume them to be active?

Reply:
Done in 8f8518c527b9c7e659fb4e149b0cc5e2f4c7dcc8